client/src/Views/Home/Prospects/Prospects.js

```diff
@@ -1,16 +1,176 @@
+          <TableCell className={this.props.classes.status} align="center">{prospect.status}</TableCell>
+          <TableCell className={this.props.classes.owner} align="center">{user.firstName + " " + user.lastName}</TableCell>
+          <TableCell className={this.props.classes.last_contacted} align="center">{"-"}</TableCell>
+          <TableCell className={this.props.classes.email_count} align="center">{0}</TableCell>
+        </TableRow>
+      ))
+    ) : 
+    (
+      //print an empty row when no prospects available - looks a bit nicer
+      <TableRow >
+          <TableCell className={this.props.classes.prospect_id} align="center"></TableCell>
+          <TableCell className={this.props.classes.email}></TableCell>
+          <TableCell className={this.props.classes.email}></TableCell>
+          <TableCell className={this.props.classes.status} align="center"></TableCell>
+          <TableCell className={this.props.classes.owner} align="center"></TableCell>
+          <TableCell className={this.props.classes.last_contacted} align="center"></TableCell>
+          <TableCell className={this.props.classes.email_count} align="center"></TableCell>
+        </TableRow>
+    );
+    
+    
+    return (
+      
+      <Route path="/home/prospects">
+            <Grid className={this.props.classes.root} container xs={12} spacing={0}>
+                <Grid item xs={3}>
+                    <ProspectSidebar
+                      onChange={this.handleFieldChange}
+                    />
+                </Grid>
+                <Grid item xs={9}>
+                  <Box className={this.props.classes.main}>
+                    <Grid
+                          className={this.props.classes.root}
+                          container
+                          direction="column"
+                          alignItems="center"
+                          wrap="nowrap"
+                    >
+                      <ProspectDashboardHeader/>
+                      <Grid
+                          item
+                          container
+                          direction="column"
+                          alignContent="center"
+                          alignItems="center"
+                          spacing={7}
+                          className={this.props.classes.root}
+                        >
+                  
+                          <Grid
+                            item
+                            container
+                            direction="column"
+                            alignContent="center"
+                            spacing={2}
+                            className={this.props.classes.list_entire}
+                          >
+                            <div className={this.props.classes.list}>
+                            <React.Fragment>
+                              <Table size="small" className={this.props.classes.table}>
```

Comment:
We'd eventually want your Table to be a reusable component since it'll also exist in the campaigns page